doc/srv6/srv6_static_config_hld.md

```diff
@@ -0,0 +1,162 @@
+# Static Configuration of SRv6 in SONiC HLD  
+
+# Table of Contents
+
+- [Revision](#revision)
+- [Definition/Abbreviation](#definitionabbreviation)
+- [About This Manual](#about-this-manual)
+- [1 Introuduction and Scope](#1-introuduction-and-scope)
+- [2 Feature Requirements](#2-feature-requirements)
+- [2.1 Functional Requirements](#21-functional-requirements)
+- [2.2 Configuration and Managment Requirements](#22-configuration-and-management-requirements)
+- [2.3 Warm Boot Requirements](#23-warm-boot-requirements)
+- [3 Feature Design](#3-feature-design)
+- [3.1 New Table in ConfigDB](#31-new-table-in-configdb)
+- [3.2 Bgpcfgd Changes](#32-bgpcfgd-changes)
+- [3.3 YANG Model](#33-yang-model)
+- [4 Unit Test](#4-unit-test)
+- [5 References ](#5-references) 
+
+# Revision
+
+| Rev  |   Date    |           Author           | Change Description      |
+| :--: | :-------: | :------------------------: | :---------------------: |
+| 0.1  | 12/5/2024 |       Changrong Wu         |  Initial version        |
+
+
+# Definition/Abbreviation
+
+### Table 1: Abbreviations
+
+| ****Term**** | ****Meaning**** |
+| -------- | ----------------------------------------- |
+| BGP  | Border Gateway Protocol |
+| SID  | Segment Identifier  |
+| SRv6 | Segment Routing IPv6  |
+| SDN | Software Defined Network |
+| uSID | Micro Segment |
+| VRF  | Virtual Routing and Forwarding  |
+
+# About this Manual
+
+This document provides general information about the design of the enhancements in SONiC to support static configuration of Segmentation Routing over IPv6 protocol, which is crucial for SRv6 SDN deployment (without usage of BGP).
```

Comment:
Are we adding support only for end-point behaviors? how about steering the traffic to SRv6 at the head-end?

Reply:
Yes. Right now, we only intend to add support for end-point behaviors because the traffic steering is done on the NICs in our use scenarios.